src/main/java/frc/robot/RobotContainer.java

```diff
@@ -74,8 +82,9 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
-    JoystickButton lightButton = new JoystickButton(this.leftJoystick, 1);
-    lightButton.whileTrue(this.m_blinkinCommand);
+    JoystickButton lightButton = new JoystickButton(this.leftJoystick, ControllerConstants.kBalanceButton);
+    lightButton.whileTrue(this.m_balancecommand);
```

Comment:
Would change name to not be light button unless the primary purpose of this button is the lights. Just reading it looks more like this button is for balancing, so the name lightButton is very misleading
```suggestion
    JoystickButton balanceButton = new JoystickButton(this.leftJoystick, ControllerConstants.kBalanceButton);
    balanceButton.whileTrue(this.m_balancecommand);
```